powertools-metrics/src/main/java/software/amazon/lambda/powertools/metrics/internal/MetricsUtils.java

```diff
@@ -0,0 +1,18 @@
+package software.amazon.lambda.powertools.metrics.internal;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import software.amazon.lambda.powertools.metrics.Metrics;
+
+import static software.amazon.lambda.powertools.common.internal.LambdaHandlerProcessor.getXrayTraceId;
+
+class MetricsUtils {
```

Comment:
Can you make this class `final` and add a private constructor such as:

```java
private MetricsUtils {
    // Utility class
}
```
?

Reply:
See Sonarlint finding and pmd analysis. 

```
For classes that only have static methods, consider making them utility classes. Note that this doesn't apply to abstract classes, since their subclasses may well include non-static methods. Also, if you want this class to be a utility class, remember to add a private constructor to prevent instantiation. (Note, that this use was known before PMD 5.1.0 as UseSingleton). UseUtilityClass (Priority: 1, Ruleset: Design) https://docs.pmd-code.org/snapshot/pmd_rules_java_design.html#useutilityclass
```